validation/mesoscale_turbulence/baroclinic_adjustment.jl

```diff
@@ -137,35 +137,33 @@ end
 simulation.callbacks[:print_progress] = Callback(print_progress, IterationInterval(20))
 
 
-slicers = (west = FieldSlicer(i=1),
-           east = FieldSlicer(i=grid.Nx),
-           south = FieldSlicer(j=1),
-           north = FieldSlicer(j=grid.Ny),
-           bottom = FieldSlicer(k=1),
-           top = FieldSlicer(k=grid.Nz))
+slicers = (west = (1, :, :),
+           east = (grid.Nx, :, :),
+           south = (:, 1, :),
+           north = (:, grid.Ny, :),
+           bottom = (:, :, 1),
+           top = (:, :, grid.Nz))
 
 for side in keys(slicers)
     field_slicer = slicers[side]
 
     simulation.output_writers[side] = JLD2OutputWriter(model, fields(model),
                                                        schedule = TimeInterval(save_fields_interval),
-                                                       field_slicer = field_slicer,
                                                        filename = filename * "_$(side)_slice",
```

Comment:
you maybe want to add to the output `indices = field_slicer`?

Reply:
I honestly don't know how this works but that sounds like a good idea.

Reply:
the `indices` keyword tells you what field "slice" to save, so in case of `indices=(:, :, grid.Nz)` you are saving only the surface